modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -3076,6 +3076,38 @@ public void testGetNestedFieldDetailsInOneToManyRelationships()
 				StringUtil.removeLast(
 					_objectDefinition1.getPKObjectFieldName(), "Id")));
 
+		_testGetNestedFieldDetailsInOneToManyRelationships(
+			StringBundler.concat(
+				_objectDefinition2.getRESTContextPath(), "?nestedFields=r_",
+				_objectRelationship1.getName(), "_",
+				StringUtil.removeLast(
+					_objectDefinition1.getPKObjectFieldName(), "Id")),
+			StringBundler.concat(
```

Comment:
Could we extract this `StringBundler.concat` to a variable?

```			
StringBundler.concat(
				"r_", _objectRelationship1.getName(), "_",
				StringUtil.removeLast(
					_objectDefinition1.getPKObjectFieldName(), "Id")
```

It is generating the same value for the 3 cases that you add, right?
Maybe in a variable, the code is more verbose?

Reply:
Sorry, I didn't see the other commits, my bad ✌️